lib/ci-stack.ts

```diff
@@ -28,15 +29,25 @@ export class CIStack extends Stack {
     const useSslParameter = new CfnParameter(this, 'useSsl', {
       description: 'If the jenkins instance should be access via SSL',
       allowedValues: ['true', 'false'],
+      default: 'false',
     });
+
+    const devModeParameter = new CfnParameter(this, 'devMode', {
+      description: 'If the jenkins instance should be access in dev mode',
```

Comment:
The description should answer the question, what am I trading off to set this flag true or false, the current comment doesn't make it very clear.